middlewares/Locale.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace RainLab\Translate\Middlewares;
+
+use RainLab\Translate\Classes\Translator;
+use Closure;
+
+class Locale
+{
+    /**
+     * Handle an incoming request.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @param  \Closure  $next
+     * @return mixed
+     */
+    public function handle($request, Closure $next)
+    {
+        $translator = Translator::instance();
+        $translator->isConfigured();
+    
+        if( !$translator->loadLocaleFromRequest() ) {
```

Comment:
That logic needed just for declare only one `Route`.
I don't care about it and create routes for all locales in the `cms.beforeRoute` event.

So i set needed locale with this code
```
 $translator = Translator::instance();

        if( !$translator->loadLocaleFromRequest() ) {
            $translator->loadLocaleFromSession();
        }
```
in `Global Middleware` — `RainLab\Translate\Middleware\Locale`